src/Sarif/Autogenerated/Run.cs

```diff
@@ -138,6 +139,8 @@ public SarifNodeKind SarifNodeKind
         /// Specifies the unit in which the tool measures columns.
         /// </summary>
         [DataMember(Name = "columnKind", IsRequired = false, EmitDefaultValue = false)]
+        [JsonConverter(typeof(EnumConverter))]
+        [System.ComponentModel.DefaultValue(ColumnKind.UnicodeCodePoints)]
```

Comment:
Jschema doesn't emit default values currently. We'll have to manually maintain them for now. [https://github.com/Microsoft/jschema/issues/57](https://github.com/Microsoft/jschema/issues/57)